nzedb/Binaries.php

```diff
@@ -615,21 +616,32 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 			}
 		}
 
-		$headersRepaired = $articles = $rangeNotReceived = $collectionIDs = $binariesUpdate = $headersReceived = $headersNotInserted = [];
+		$headersRepaired = $mgrHeadersRepaired = $articles = $rangeNotReceived = $mgrRangeNotReceived = $collectionIDs = $binariesUpdate = $headersReceived = $headersNotInserted = $mgrHeadersReceived = $mgrHeadersNotInserted = [];
 		$notYEnc = $headersBlackListed = 0;
 
 		$partsQuery = $partsCheck = sprintf('INSERT IGNORE INTO %s (binaryid, number, messageid, partnumber, size) VALUES ', $tableNames['pname']);
+		$mgrPartsQuery = $mgrPartsCheck = sprintf('INSERT IGNORE INTO mgr_parts (binaryid, number, messageid, partnumber, size) VALUES ');
 
 		$this->_pdo->beginTransaction();
 		// Loop articles, figure out files/parts.
 		foreach ($headers as $header) {
 
+			ReleasesMultiGroup::isMultiGroup($header['From']) ? $multiGroup = true : $multiGroup = false;
```

Comment:
Intent is to set a flag for MGR so second set of tables is used for MG releases.